lib/wings/valkyrie/query_service.rb

```diff
@@ -77,6 +77,24 @@ def find_by_alternate_identifier(alternate_identifier:)
         raise ::Valkyrie::Persistence::ObjectNotFoundError
       end
 
+      # Find all members of a given resource, and map to Valkyrie Resources
+      # @param [Valkyrie::Resource]
+      # @param [Valkyrie::ResourceClass]
+      # @return [Array<Valkyrie::Resource>]
+      def find_members(resource:, model: nil)
+        find_model = model.internal_resource.constantize if model
```

Comment:
I think it's minor, but as I was reading through the code I felt like it could benefit with chaining a select to a map which seemed somewhat easier to read.. 

    member_list = resource_factory.from_resource(resource: resource).try(:members)
    return [] unless member_list

    member_list.select { |obj| model ? obj.class == find_model : obj }
               .map { |obj| resource_factory.to_resource(object: obj) }